### PR TITLE
[network/go] fixup inv -e system-probe.generate-lookup-tables

### PR DESCRIPTION
**Pull request by nplanel**
### What does this PR do?

Looking at our QA card for https://github.com/DataDog/datadog-agent/pull/10039 https://github.com/DataDog/datadog-agent/pull/12897
Found that inv -e system-probe.generate-lookup-tables gotls/lookup table generation was working correctly

It's a minor issue not affecting 7.38 runtime as the generated files are the same

original error
`
go generate ./pkg/network/http/gotls/lookup/main.go
package command-line-arguments
	imports github.com/DataDog/datadog-agent/pkg/network/go/bininspect: build constraints exclude all Go files in /home/nplanel/dd/datadog-agent/pkg/network/go/bininspect
`

files containing build linux_bpf tag dependency
* `pkg/network/go/bininspect/inspect.go` 
* `pkg/network/http/gotls/lookup/luts.go`

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Comment by brycekahle** [comment]
@nplanel I think this can wait for 7.40 since it doesn't affect any production code.

File: pkg/network/go/goid/goid_offset.go [hunk 1]
```diff
@@ -1,5 +1,8 @@
 // Code generated by go generate; DO NOT EDIT.
 
+//go:build linux_bpf
+// +build linux_bpf
```
Comment (brycekahle) [inline pkg/network/go/goid/goid_offset.go:4]: Why does the output need these build tags? We don't use ebpf for the goid offset.

Comment (nplanel) [reply pkg/network/go/goid/goid_offset.go:4]: `pkg/network/http/gotls/lookup/luts.go` having the build tag dependency already commited in the repo (generated gotls/lookup/main => generate_luts.go) 

but yeah may we should avoid tag pollution

File: pkg/network/http/gotls/lookup/main.go [hunk 2]
```diff
@@ -9,4 +9,4 @@ package lookup
 // - Use /var/tmp/datadog-agent/system-probe/go-toolchains
 //   as the location for the Go toolchains to be downloaded to.
 //   Each toolchain version is around 500 MiB on disk.
-//go:generate go run ./internal/generate_luts.go --test-program ./internal/program.go --package lookup --out ./luts.go --min-go 1.13 --arch amd64,arm64 --shared-build-dir /var/tmp/datadog-agent/system-probe/go-toolchains
+//go:generate go run -tags linux_bpf ./internal/generate_luts.go --test-program ./internal/program.go --package lookup --tags linux_bpf --out ./luts.go --min-go 1.13 --arch amd64,arm64 --shared-build-dir /var/tmp/datadog-agent/system-probe/go-toolchains
```
Comment (brycekahle) [inline pkg/network/http/gotls/lookup/main.go:12]: Is the `-tags linux_bpf` (the single dash version) needed here?

Comment (nplanel) [reply pkg/network/http/gotls/lookup/main.go:12]: yes because it will not be able to compile pkg/network/go/bininspect/

Comment (nplanel) [reply pkg/network/http/gotls/lookup/main.go]: yeah it's not right, it's seems the issue bpf_linux tags dependency in pkg/network/go and shouldn't been set in these directort/subdirs

**Review by brycekahle — APPROVED** [review]

**Pull request merged**